include/rdma/fi_domain.h

```diff
@@ -179,6 +179,7 @@ struct fi_mr_attr {
 		int		ze;
 		int		neuron;
 		int		synapseai;
+		int		rocr;
```

Comment:
This is a core API change so need @j-xiong to review... I am surprised that we didn't have that earlier